uni/lib/view/useful_info/useful_info.dart

```diff
@@ -0,0 +1,44 @@
+import 'package:flutter/material.dart';
+import 'package:uni/view/common_widgets/pages_layouts/general/general.dart';
+import 'package:uni/view/useful_info/widgets/academic_services_card.dart';
+import 'package:uni/view/useful_info/widgets/copy_center_card.dart';
+import 'package:uni/view/useful_info/widgets/dona_bia_card.dart';
+import 'package:uni/view/useful_info/widgets/infodesk_card.dart';
+import 'package:uni/view/useful_info/widgets/multimedia_center_card.dart';
+import 'package:uni/view/useful_info/widgets/other_links_card.dart';
+import 'package:uni/view/useful_info/widgets/sigarra_links_card.dart';
+import 'package:uni/view/common_widgets/page_title.dart';
+
+class UsefulInfoCardView extends StatefulWidget {
+  const UsefulInfoCardView({super.key});
+
+  @override
+  State<StatefulWidget> createState() => UsefulInfoCardViewState();
+}
+
```

Comment:
Because this is a page and not a card I suggest to rename it to:

`UsefulInfoCardView` -> `UsefulInfoPageView` or `UsefulInfoPage`
`UsefulInfoCardViewState` -> `UsefulInfoPageViewState` or `UsefulInfoPageState`